cli/src/commands/extensions/api.rs

```diff
@@ -151,7 +164,7 @@ struct ProcessOutput {
 #[serde]
 async fn analyze(
     op_state: Rc<RefCell<OpState>>,
-    #[serde] packages: Vec<PackageWithOrigin>,
+    #[serde] packages: Vec<ExtensionApiPackage>,
```

Comment:
Is this change worthy of a `CHANGELOG.md` entry?

Reply:
Yes we should state that purl is now a supported format.